packages/inference/src/HfInference.ts

```diff
@@ -680,13 +680,14 @@ export class HfInference {
 		const mergedOptions = { ...this.defaultOptions, ...options };
 		const { model, ...otherArgs } = args;
 
-    const headers = {
-      Authorization: `Bearer ${this.apiKey}`,
-    }
+		const headers = {
+			Authorization:  `Bearer ${this.apiKey}`,
+			"Content-Type": "application/json",
```

Comment:
You can avoid setting this only if `options?.binary` - this should also answer your question :)